lib/calendar_urls.coffee

```diff
@@ -0,0 +1,83 @@
+ADDRESS_ATTR = ''
+TITLE_ATTR = ''
+
+module.exports = ({ address_attr, title_attr }) ->
+  ADDRESS_ATTR = address_attr
+  TITLE_ATTR = title_attr
+  module.exports.methods
+
+module.exports.methods =
+  formatTime: formatTime = (date) ->
+    new Date(date).toISOString().replace(/-|:|\.\d+/g, '')
+
+  calulateDurationTime: calulateDurationTime = (end, start) ->
```

Comment:
Neat way to hoist a local variable here! `calulateDurationTime: calulateDurationTime =`